libs/src/services/creatorNode/index.js

```diff
@@ -187,7 +187,8 @@ class CreatorNode {
         method: 'post',
         data: {
           metadata,
-          blockNumber
+          blockNumber,
+          forceWrite
```

Comment:
is this field used anywhere in the post route? I dont see a change in that route in this PR

Reply:
middleware

Reply:
o doi!